tests/riotboot/README.md

```diff
@@ -17,5 +17,24 @@ This test should foremost give you an overview how to use riotboot:
 In this test two modules `riotboot_hdr` and `riotboot_slot` are used to showcase
 the access to riotboot shared functions.
 
-  - `make test` can be executed to run the automatic Python test that checks
-  basic functionalities of riotboot
+Automatic test
+==============
+
+This application's "test" target can be used to test basic riotboot
+functionality:
+
+    BOARD=<board> make flash test
```

Comment:
This only creates the first slot (`slot0`) so the test fails because it can't find `slot1`. Can we change default behavior so it builds both slots?

Also how does this work in the ci? I guess it is using an old version of slot1? Shouldn't previous ones?

Reply:
Ah, I missed that.
For the CI, the target "make test-murdock" depends on "$(TEST_EXTRA_FILES)". We might have to add those also to the test target.

Reply:
I've added the TEST_EXTRA_FILES as dependencies to the "make test" target. Could you try again?

Reply:
nope, something is still off.

Reply:
Ok, I solved it by removing the TEST_EXTRA_FILES dependencies from both "make test" and "make test-murdock", but adding that both slot elf files are created in the link step.

@fjmolinas please take another look!

Reply:
Now it is working fine.

Reply:
I'm trying to figure out if the are any impacts of linking both by default. When compiling I get no hint that both where linked, but since `*slot1.elf` was linked if afterwards flash `slot1` I could incorrectly assume that it will recompile form scratch to make `slot1`. I don't mind this behavior but I think the user should now of it, probably threw some compiler output. What do you think?

Reply:
@kaspar030 ping!